ghost/core/core/server/data/migrations/versions/6.0/2025-06-30-14-00-00-update-feature-image-alt-length.js

```diff
@@ -0,0 +1,26 @@
+const logging = require('@tryghost/logging');
+const {createNonTransactionalMigration} = require('../../utils');
+const DatabaseInfo = require('@tryghost/database-info');
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
+        if (DatabaseInfo.isSQLite(knex)) {
+            logging.warn('Skipping migration for SQLite3');
+            return;
+        }
```

Comment:
_Very_ minor thing but seeing as I had to look up why we skip for SQLite I think we should start commenting these blocks with the reason for skipping. In this case it seems it's because SQLite doesn't have fixed-length varchar columns?